### PR TITLE
13.0 mig pos access right

### PR DESCRIPTION
**Pull request by ammarofficewla**

### CODE REVIEW

**Review by eduardgm — APPROVED** [review]
Functional review 👍

LGTM

File: pos_access_right/static/src/js/pos_access_right.js [hunk 1]
```diff
@@ -13,10 +13,28 @@ odoo.define('pos_access_right.pos_access_right', function (require) {
...
 
+
+
     // New function 'display_access_right' to display disabled functions
+    gui.Gui.prototype.get_user_groups = function (){
+        var self = this;
+        if (this.pos.get_cashier() && this.pos.user && this.pos.get_cashier().user_id[0] == this.pos.user.id) {
+            return this.pos.user
+        }else{
+           return  _.find(this.pos.users, function(user_id){
```
Comment (ivantodorovich) [inline pos_access_right/static/src/js/pos_access_right.js:27]:
```suggestion
           return _.find(this.pos.users, function(user_id){
```

**Review by danimv5 — APPROVED** [review]
Functional review! LGTM

**Review by chienandalu — CHANGES_REQUESTED** [review]
I'm afraid pre-commit is going to be needed now for this to be merged: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#howto

**Pull request closed without merging**